### DISCUSSION TITLE
Add additional LibreOffice languages to Deepin ISO

### DISCUSSION THREAD

**Discussion by eduardomozart** [linuxdeepin/developer-center#3349] [Features Request & Ideas | 特性请求 & 头脑风暴]
After installing Deepin 20.6, I had to manually install the ``libreoffice-l10n-pt-br`` package to translate LibreOffice to my native language. As Deepin is a multi language operating system, it should bring language pack packages to LibreOffice for every language it's support, to make it easiers to new users to start using Deepin instead of troubleshooting this kind of issue.

**Comment by BLumia** [comment]
IMO it will be a lot of extra packages installed for people who only speak one or two languages and some people really want to avoid installing unnecessary packages if possible.

**Comment by eduardomozart** [comment]
I see your point, but the packages could still be provided at the ISO and be installed based on the user language selection. The average user that is starting to use Linux for the first time will see LibreOffice in English and will not know what to do.

**Reply by BLumia** [reply]
cc @meiyixiang 需求：deepin 非中文语言安装时，预装 libreoffice 时希望能根据所使用的语言安装对应的翻译语言包。

**Comment by meiyixiang** [comment]
Take your advice

**Comment by meiyixiang** [comment]
We need to evaluate how we're going to do this, because in the future, if we have a lot of applications, how are we going to support different languages.
